packages/field-base/src/keyboard-mixin.d.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+/**
```

Comment:
nit: please add a newline to separate these two JSDoc comments.

Reply:
Done.